src/ui/components/atoms/Primary-Button/PrimaryButton.tsx

```diff
@@ -0,0 +1,19 @@
+import styles from './PrimaryButton.module.scss';
+import type { IProps } from './types/IProps';
+import Add from '../../../../assets/icons/add.svg?raw';
+
+function PrimaryButton(props: IProps) {
+  const { children, variant, onClick, icon } = props;
+  const selectedIcon = icon ?? (children === 'Primary button' ? Add : Add);
+  return (
+    <button
+      className={`${styles['primary-button']} ${styles[`primary-button--${variant}`]}`}
```

Comment:
Nice pr, really good job, only a few things, please use this approach https://github.com/Ditmar/backoffice-openscience/blob/master/src/ui/components/demo/calculator/atoms/display.tsx to create your variants of the components.

Reply:
![image](https://github.com/user-attachments/assets/0c27aea1-a977-4435-bf17-a80a531965bc)
ya lo corregi ing

Reply:
![image](https://github.com/user-attachments/assets/4ab42e97-13be-4249-b72e-26e6a6ab57c1)

Reply:
I request a review of my PR:
https://ditmar.github.io/backoffice-openscience/branch-feature/SEM-006/?path=/story/ui-components-atoms-primary-button